dscensor/setup.py

```diff
@@ -0,0 +1,22 @@
+# setup.py
+from setuptools import find_packages, setup
+
+from dscensor import __version__
+
+setup(
+    name="dscensor",
+    version=__version__,
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
```

Comment:
This is redundant since `setup.cfg` (which pip-tools parses) already defines these requirements.